packages/flutter/lib/src/rendering/object.dart

```diff
@@ -2205,13 +2205,12 @@ abstract class RenderObject with DiagnosticableTreeMixin implements HitTestTarge
   /// This property can also be null while an ancestor in the tree is
   /// currently doing layout, until this render object itself does layout.
   ///
-  /// When not null, the relayout boundary is either this render object itself
-  /// or one of its ancestors, and all the render objects in the ancestry chain
-  /// up through that ancestor have the same [_relayoutBoundary].
-  /// Equivalently: when not null, the relayout boundary is either this render
-  /// object itself or the same as that of its parent.  (So [_relayoutBoundary]
-  /// is one of `null`, `this`, or `parent!._relayoutBoundary!`.)
-  RenderObject? _relayoutBoundary;
+  /// When [_isRelayoutBoundary] is false, then `parent?._isRelayoutBoundary`
+  /// may be true or false but not null.  As a result, when it's known that
```

Comment:
nit: remove double space after `.`

Reply:
Also: The last sentence in this paragraph reads somewhat awkward. Maybe that can be rephrased?